cmd/metal-api/internal/grpc/grpcServer.go

```diff
@@ -0,0 +1,191 @@
+package grpc
```

Comment:
Can we name this file just `server.go`? Otherwise we will introduce "camelCase" for file names together with "hyphenation".

Reply:
Indeed, probably we should agree on either camelCase, hyphonetation-style or under_scoring, since their are all used in distinct metal-stack places.
Personally I pretty much prefer camelCase, since it is shorter, faster to type and even better readable.
Additionally, tests are more clearly identifiable by just looking at the contents of a golang package.

Reply:
Besides this, I would leave the grpc prefix as is, since you can find it easier from within your IDE or via rg.
Also, it doesn't affect the golang code at all.
Nevertheless, the fqn of the server struct itself is in fact `grpc.Server`